package.json

```diff
@@ -34,9 +34,9 @@
     "ocis-kill": "killall ocis",
     "docker-kill": "docker kill phoenix-tests-selenium phoenix-tests-redis",
     "testing-app": "if [ -d tests/testing-app ]; then (cd tests/testing-app && git pull && pwd); else git clone --depth 1 https://github.com/owncloud/testing.git tests/testing-app; fi;",
-    "build-ocis": "if [ -d tests/ocis ]; then (cd tests/ocis && git reset --hard master && git pull); else git clone --depth 1 https://github.com/owncloud/ocis.git tests/ocis; fi; cd tests/ocis; make clean; make",
-    "ocis": "yarn run build-ocis; REVA_STORAGE_OWNCLOUD_REDIS_ADDR='localhost:6379' REVA_STORAGE_HOME_EXPOSE_DATA_SERVER=1 REVA_STORAGE_OC_EXPOSE_DATA_SERVER=1 PHOENIX_ASSET_PATH='./dist' tests/ocis/bin/ocis server &",
-    "ocis:mac": "yarn run build-ocis; REVA_STORAGE_OWNCLOUD_REDIS_ADDR='localhost:6379' PROXY_OIDC_ISSUER='https://host.docker.internal:9200' KONNECTD_INSECURE='true' KONNECTD_IDENTIFIER_REGISTRATION_CONF='./tests/acceptance/mac-identifier-registration.yml' KONNECTD_ISS='https://host.docker.internal:9200' KONNECTD_TLS='true' PHOENIX_WEB_CONFIG='./tests/acceptance/ocis-mac-config.json' PHOENIX_ASSET_PATH='./dist' bin/ocis server &",
+    "build-ocis": "if [ -d tests/ocis ]; then (cd tests/ocis && git reset --hard master && git pull); else git clone --depth 1 https://github.com/owncloud/ocis.git tests/ocis; fi; cd tests/ocis/ocis; make clean; make",
+    "ocis": "yarn run build-ocis; STORAGE_HOME_DRIVER=owncloud STORAGE_USERS_DRIVER=owncloud REVA_STORAGE_OWNCLOUD_REDIS_ADDR='localhost:6379' REVA_STORAGE_HOME_EXPOSE_DATA_SERVER=1 REVA_STORAGE_OC_EXPOSE_DATA_SERVER=1 PHOENIX_ASSET_PATH='./dist' tests/ocis/ocis/bin/ocis server &",
+    "ocis:mac": "yarn run build-ocis; STORAGE_HOME_DRIVER=owncloud STORAGE_USERS_DRIVER=owncloud REVA_STORAGE_OWNCLOUD_REDIS_ADDR='localhost:6379' PROXY_OIDC_ISSUER='https://host.docker.internal:9200' KONNECTD_INSECURE='true' KONNECTD_IDENTIFIER_REGISTRATION_CONF='./tests/acceptance/mac-identifier-registration.yml' KONNECTD_ISS='https://host.docker.internal:9200' KONNECTD_TLS='true' PHOENIX_WEB_CONFIG='./tests/acceptance/ocis-mac-config.json' PHOENIX_ASSET_PATH='./dist' bin/ocis/ocis server &",
```

Comment:
Quick note: `PHOENIX_WEB_CONFIG` and `PHOENIX_ASSET_PATH` are not working so this, unfortunately, won't work yet as it will assume the server is still `localhost` https://github.com/owncloud/ocis/issues/672 (that's why I delayed my PR where I wanted to change this).

Reply:
@LukasHirt is that fixed by https://github.com/owncloud/ocis/pull/808 ?

Reply:
Not sure if it's because of that PR but yes, it is working now again.

Reply:
@LukasHirt please review this PR then

Reply:
Will test it today in the afternoon :wink: